english/starfish/starfish.md

```diff
@@ -0,0 +1 @@
+You think, "it's not easy being green." Your starfish mother tells you about a long line of green starfish ancestors that have achieved great things and now it's your turn to make your mark. You feel proud to be green now. You won't let the green starfish family down.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.